packaging/debs/Debian/debian/rules

```diff
@@ -49,8 +50,9 @@ override_dh_auto_install:
 		$(DEB_DESTDIR)$(PREFIX)/lib/ocf/resource.d/rabbitmq/rabbitmq-server
 	install -p -D -m 0755 scripts/rabbitmq-server-ha.ocf \
 		$(DEB_DESTDIR)$(PREFIX)/lib/ocf/resource.d/rabbitmq/rabbitmq-server-ha
-	install -p -D -m 0644 scripts/set_rabbitmq_policy.sh \
-		$(DEB_DESTDIR)$(PREFIX)/lib/ocf/resource.d/rabbitmq/set_rabbitmq_policy.sh.example
+
+	install -p -D -m 0644 docs/set_rabbitmq_policy.sh.example \
+		$(DEB_DESTDIR)$(RMQ_DOCS_DIR)/set_rabbitmq_policy.sh.example
```

Comment:
This file should be listed in `debian/rabbitmq-server.docs` instead of being manually installed by `debian/rules`.